products/red-hat-openshift.md

```diff
@@ -16,21 +16,29 @@ eolColumn: Maintenance Support
 extendedSupportColumn: Extended Update Support
 
 releases:
+-   releaseCycle: "4.13"
+    releaseDate: 2023-05-17
+    support: true
+    eol: 2024-11-17
+    extendedSupport: 2025-01-17
```

Comment:
Shouldn't this field only set to `true` until the date is known ?

Reply:
That is correct. I added a comment for future update. What do you think?